code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -377,7 +378,8 @@
 		/obj/item/gun/energy/disabler/cyborg,
 		/obj/item/melee/transforming/energy/sword/cyborg,
 		/obj/item/gun/energy/pulse/carbine/cyborg,
-		/obj/item/clothing/mask/gas/sechailer/cyborg)
+		/obj/item/clothing/mask/gas/sechailer/cyborg,
+		/obj/item/weldingtool/cyborg)
```

Comment:
why does engiborg have two welders?

Reply:
![image](https://user-images.githubusercontent.com/22431091/189371227-edabb074-ff28-4a94-aa00-684ef64dcb44.png)

No?

Reply:
Bleh, misread the code